src/main/kotlin/org/jitsi/nlj/transform/node/incoming/BitrateCalculator.kt

```diff
@@ -67,3 +73,25 @@ class VideoBitrateCalculator(
 
     override fun trace(f: () -> Unit) = f.invoke()
 }
+
+open class BitrateCalculator(name: String = "Bitrate calculator") : ObserverNode(name) {
+    private val rateStatistics = RateStatistics(5000, 8000f)
+    val bitrate: Bandwidth
+        get() = rateStatistics.rate.bps
```

Comment:
Is there any desire to keep a packet rate as well as a bitrate here?

Reply:
I can't think of any uses for it right now, and it would be trivial to add if/when the need arises.

Reply:
I was actually wondering if "receiving packets" would be a better proxy for "is sending" than "receiving bits" - though it'd be hard for audio to drop below 10 kbps either way, since we're counting RTP headers.

Reply:
I found one corner case: screensharing static content. With a relatively small resolutionI see 5 pps, 5 kbps.

Reply:
Sounds like pps (maybe in the 1-2 range) would be a better metric, then?